easybuild/framework/easyconfig/templates.py

```diff
@@ -299,6 +300,14 @@ def template_constant_dict(config, ignore=None, skip_lower=None, toolchain=None)
             # don't fail just because we couldn't resolve this template
             _log.warning("Failed to create mpi_cmd_prefix template, error was:\n%s", err)
 
+    # step 6. CUDA compute capabilities
+    cuda_compute_capabilities = build_option('cuda_compute_capabilities') or config.get('cuda_compute_capabilities')
+    if cuda_compute_capabilities:
+        template_values['cuda_compute_capabilities'] = ','.join(cuda_compute_capabilities)
+        sm_values = ['sm_' + cc.replace('.', '') for cc in cuda_compute_capabilities]
+        template_values['cuda_sm_list'] = ','.join(sm_values)
+        template_values['cuda_sm_values'] = ' '.join(sm_values)
```

Comment:
@Flamefire How about renaming these to `cuda_sm_comma_sep` and `cuda_sm_space_sep`?

Reply:
Good idea. Done